serverlessworkflow/modules/ROOT/pages/core/timeouts-support.adoc

```diff
@@ -0,0 +1,388 @@
+
+==== PostgreSQL
+
+PostgreSQL is the recommended database to use with job service**.**
+Additionally, it provides an initialization procedure that integrates link:https://flywaydb.org[Flyway] for the database initialization. It automatically controls the database schema, in this way all tables are created by the service.
+
+In case you need to externally control the database schema, you can check the Flyway SQL scripts in link:{kogito_apps_url}/jobs-service/jobs-service-postgresql/src/main/resources/db/migration[migration] and apply them.
+
+You need to set the proper configuration parameters when starting job service.
+The example shows how to run PostgreSQL as a Kubernetes deployment, but you can run it the way it fits in your environment, the important part is to set all the configuration parameters points to your running instance of PostgreSQL.
+
+==== Ephemeral
+Alternatively, there is an in-memory database support that does not require any external database configuration, it can be used for testing and development purposes, but it is not recommended for production, since all jobs are lost upon a service restart or failure.
+
+=== Job Service communication
+
+[NOTE]
+====
+The Job Service does not execute a job but triggers a callback that might be an HTTP request or a Cloud Event that is managed by the configured jobs addon see(link addons config) in the workflow application.
+====
+
+==== Knative Eventing
+
+To configure the communication between the Job Service and the workflow runtime through the knative eventing system, you must provide a set of configurations.
+
+The Job Service configuration is done through the deployment descriptor shown in the <<job-service-deploy, example>>.
+
+==== Addon configuration in the workflow runtime
+
+The communication from the workflow application with Job Service is done through an addon, which is responsible for publishing and consuming events related to timers.
+When you run the workflow as a knative service, you must add the `kogito-addons-quarkus-jobs-knative-eventing` to your project and provide the proper configuration.
+
+* Dependency in the `pom.xml`:
+
+.Callback state example with timeout
+[source, xml]
+----
+<dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-addons-quarkus-jobs-knative-eventing</artifactId>
+</dependency>
+----
+
+* Configuration parameters:
+
+.Callback state example with timeout
+[source, properties]
+----
+# Events produced by kogito-addons-quarkus-jobs-knative-eventing to program the timers on the Job Service.
+mp.messaging.outgoing.kogito-job-service-job-request-events.connector=quarkus-http
+mp.messaging.outgoing.kogito-job-service-job-request-events.url=${K_SINK:http://localhost:8280/jobs/events}
+mp.messaging.outgoing.kogito-job-service-job-request-events.method=POST
+----
```

Comment:
I think it's worth explaining that the `K_SINK` variable is injected by Knative Eventing and add alink here to our Knative Eventing integration guide.

Reply:
+1